subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/configurations/DefaultConfiguration.java

```diff
@@ -1534,6 +1525,34 @@ private void assertIsResolvable() {
         }
     }
 
+    private void assertIsDeclarable() {
+        if (!canBeDeclared) {
+            throw new IllegalStateException("Declaring dependencies for configuration '" + name + "' is not allowed as it is defined as 'canBeDeclared=false'.\n");
```

Comment:
Seems weird we're adding a newline to the end of an exception message. Is there a reason you did this?

Reply:
Just a copy/paste mistake.